includes/admin/class-pass-manager.php

```diff
@@ -83,28 +90,30 @@ public function __construct() {
 			$this->has_pass_data = true;
 		}
 
-		$this->highest_pass_id = $this->get_highest_pass_id();
+		// Set up the highest pass data.
+		$this->set_highest_pass_data();
 
 		global $edd_licensed_products;
 
 		$this->number_license_keys = is_array( $edd_licensed_products ) ? count( $edd_licensed_products ) : 0;
 	}
 
 	/**
-	 * Returns the ID of the highest tier pass that's activated.
+	 * Gets the highest pass and defines its data.
 	 *
-	 * @since 2.10.6
-	 *
-	 * @return int|null Pass ID if one is found, null if one was not found.
+	 * @since 2.11.x
+	 * @return void
 	 */
-	private function get_highest_pass_id() {
-		$highest_pass_id = null;
+	private function set_highest_pass_data() {
 
 		if ( ! $this->has_pass_data || ! is_array( $this->pass_data ) ) {
-			return $highest_pass_id;
+			return;
 		}
 
-		foreach ( $this->pass_data as $pass_data ) {
+		$highest_license_key = false;
+		$highest_pass_id     = false;
```

Comment:
```suggestion
		$highest_license_key = null;
		$highest_pass_id     = null;
```

I'd prefer null defaults for these.